digest.cfg

```diff
@@ -18,8 +18,15 @@ medium_access_token:
 output_file_name_pattern: {author}_{msid:0>5}.docx
 # DOI formatting
 doi_pattern: 
+# IIIF image server settings
+iiif_image_uri:
+iiif_image_source_uri:
+iiif_info_url: 
 
 [elife]
 medium_license: cc-40-by
-medium_image_url: https://iiif.elifesciences.org/digests:{msid:0>5}%2F{file_name}/full/full/0/default.jpg
+medium_image_url: https://iiif.elifesciences.org/digests/{msid:0>5}%2F{file_name}/full/full/0/default.jpg
 doi_pattern: https://doi.org/10.7554/eLife.{msid:0>5}
+iiif_image_uri: https://iiif.elifesciences.org/digests/{msid:0>5}%2F{file_name}
```

Comment:
`digests/` is the correct IIIF prefix, see description of https://github.com/elifesciences/issues/issues/4310 (I had doubts)